nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -642,6 +649,8 @@ def add_electrical_series(
     else:
         raise NotImplementedError(f"iterator_type ({iterator_type}) should be either 'v1' or 'v2' (recommended)!")
 
+    # Not sure if recovering the gains from the parent recording should be done here AFTER the data is extracted.
+    
```

Comment:
As discussed, maybe the extraction of gains should be here. Maybe it is working in our case because `BinDatRecorder` does not really use `return_scaled`.